src/store/reducers.ts

```diff
@@ -3,19 +3,35 @@
  */
 
 import { combineReducers } from '@reduxjs/toolkit'
-import { InjectedReducersType } from 'utils/types/injector-typings'
-import { createHashHistory, createBrowserHistory } from 'history'
-import { connectRouter } from 'connected-react-router'
-
-export const history = process.env.EXTENSION ? createHashHistory() : createBrowserHistory()
+import createWalletReducer from 'app/pages/CreateWalletPage/slice'
+import openWalletReducer from 'app/pages/OpenWalletPage/slice'
+import accountReducer from 'app/state/account'
+import fatalErrorReducer from 'app/state/fatalerror'
+import ledgerReducer from 'app/state/ledger'
+import networkReducer from 'app/state/network'
+import stakingReducer from 'app/state/staking'
+import transactionReducer from 'app/state/transaction'
+import walletReducer from 'app/state/wallet'
+import themeReducer from 'styles/theme/slice'
 
 /**
  * Merges the main reducer with the router state and dynamically injected reducers
  */
-export function createReducer(injectedReducers: InjectedReducersType = {}) {
+export function createReducer(injectedReducers = {}) {
   // Initially we don't have any injectedReducers, so returning identity function to avoid the error
-  return combineReducers({
+  const rootReducer = combineReducers({
     ...injectedReducers,
-    router: connectRouter(history),
+    account: accountReducer,
+    createWallet: createWalletReducer,
+    fatalError: fatalErrorReducer,
+    ledger: ledgerReducer,
```

Comment:
RootState type can now be non-optional
https://github.com/oasisprotocol/oasis-wallet-web/blob/5cebb40d55c0a51badaf23f46a9303781453e5a3/src/types/RootState.ts#L14-L23